lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -100,21 +99,43 @@ def cmd_list_data_services()
           end
 
           def cmd_add_data_service(*args)
+            protocol = "http"
             while (arg = args.shift)
               case arg
+                when '--help'
```

Comment:
The help flag is inconsistent with other commands. Perhaps we should make -h and --help aliases for help and make the hostname/address a positional argument.

Reply:
Yeah I'm ok with this. I'll make those changes.